unityfiles/Assets/Scripts/Helper/Letter_gen_Finn_pos/letter_sett_lists/letter_frequensy_weight_100.json

```diff
@@ -0,0 +1,58 @@
+{
+"letter": [
+"A",
+"B",
+"C",
+"D",
+"E",
+"F",
+"G",
+"H",
+"I",
+"J",
+"K",
+"L",
+"M",
+"N",
+"O",
+"P",
+"Q",
+"R",
+"S",
+"T",
+"U",
+"V",
+"W",
+"X",
+"Y",
+"Z"
+],
+"weight": [
+81.67,
+96.59,
+124.41,
+166.94,
+293.96,
+316.24,
+336.39,
+397.33,
+466.99,
+468.52,
+476.24,
+516.49,
+540.55,
+608.04,
+683.1099999999999,
+702.3999999999999,
+703.3499999999999,
+763.2199999999999,
+826.4899999999999,
+917.0499999999998,
+944.6299999999999,
+954.4099999999999,
+978.0099999999999,
+979.5099999999999,
+999.2499999999999,
```

Comment:
Vi trenger vel neppe så presis vekting

Reply:
Tror kanskje det skaper mer problemer for oss når vi skal begynne og legge dette sammen og lage leaderboard og slikt

Reply:
ble slik pga justering av mot den ønskede rekkevidden. Det er heller ikke en verdi table men en random choise tilpasset table for økt hastighet under valg